api/app/reviews/tests.py

```diff
@@ -1465,27 +1439,47 @@ def static_seed_data(self):
         db.session.commit()
 
         self.first_headers = self.get_auth_header_for("firstuser@mail.com")
-        # self.other_headers = self.get_auth_header_for("someuser@mail.com")
 
         db.session.flush()
 
     def test_get_reference_request_by_response_id(self):
+        self.seed_static_data()
+        reference_req = ReferenceRequest(1, 'Mr', 'John', 'Snow', 'Supervisor', 'common@email.com')
+        reference_request_repository.create(reference_req)
+        REFERENCE_DETAIL = {
+            'token': reference_req.token,
+            'uploaded_document': 'DOCT-UPLOAD-78999',
+        }
+        # response = self.app.get(
+        #     '/api/v1/reference', data=REFERENCE_DETAIL, headers=self.first_headers)
+        # self.assertEqual(response.status_code, 201)
+
+        params = {'event_id': 1, 'application_form_id': 1, 'user_id:': 5, 'sort_column' : 'review_response_id',
+                  'reference_detail': REFERENCE_DETAIL} # confirm params, check ids

```

Comment:
I actually have no idea what I'm doing here. This is a produce of copy paste :) 
I realised I wasn't entirely sure how "params" works usually - if there's flexibility in what can be added?

Does referencereq.token handle the linking of the ReferenceRequest and Reference pairs?

Reply:
So params is all of the inputs that the particular API you're calling expects. In this case you're calling the review API get method and it uses a [request parser](https://github.com/deep-learning-indaba/Baobab/blob/732dd643a99dc46fe99e7211f1e3c3ee71a9ce14/api/app/reviews/api.py#L114) which is defined [here](https://github.com/deep-learning-indaba/Baobab/blob/732dd643a99dc46fe99e7211f1e3c3ee71a9ce14/api/app/reviews/mixins.py#L3)  and that specifies what inputs the API expects. In this case it's just `event_id` which is required and `skip` which is optional

Reply:
I completely forgot about the request_parser ... but, I understand better now how it fits in!
That obviously makes way more sense than what I was thinking - everything links back to the event_id